bubblewrap.c

```diff
@@ -2419,6 +2419,10 @@ parse_args_recurse (int          *argcp,
           unsigned long long size;
           char *endptr = NULL;
 
+          if (is_privileged)
+            die ("Cowardly refusing to set tmpfs --size in privileged mode. "
+                 "Open an issue if you need this.");
```

Comment:
I'd prefer to phrase this something like:

```
The --size option is not permitted in setuid mode
```

similar to how we report `--userns`, `--userns2`.